src/docs/daily/2024-12-18.md

```
@@ -0,0 +1,23 @@
+- [Slime.kt](../../main/kotlin/day03/Slime.kt) - 슬라임(Slime) 클래스
+- [Cleric.kt](../../main/kotlin/day03/Cleric.kt) - 성직자(Cleric) 클래스
+- [ClericTest.kt](../../test/kotlin/ClericTest.kt) - 성직자(Cleric) 테스트 코드
+
+## 🔍 참고 자료
+
+- [Test code using JUnit in JVM – tutorial](https://kotlinlang.org/docs/jvm-test-using-junit.html)
+
+## ❓ 궁금한 점
+
+- TDD
+    - `Cleric`클래스를 TDD로 짰다면 더 효율적이었을까?
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)